mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -426,6 +426,20 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
                 />
             </div>
             <div className="chat-messages">
+                {displayOptimizedChatHistory.length <= 1 &&
+                    <div className="chat-empty-message">
+                        <p>
+                            Ask your personal Python expert anything! 
+                            <br />
+                            Hint: 
+                            {[
+                                " Use @ to reference variables.",
+                                " Use CTRL + E to chat with Mito AI.",
```

Comment:
The spacing of this text looks great on wide task panes, 
![Screenshot 2024-12-19 at 11 51 43 AM](https://github.com/user-attachments/assets/dde5c044-6764-4c6b-a00c-61f8868d4c05)

But a little bit awkward on narrower screens. 
![Screenshot 2024-12-19 at 11 51 55 AM](https://github.com/user-attachments/assets/3822cfb9-ddb0-4a98-8a4f-c8ecaf26ee54)

Any easy wins here you can think of?

Reply:
Ohh also, instead of hardening CTRL + E, we should make sure that we're using the right keyboard shortcut for the OS

Reply:
Adding a long and short message to deal with narrow screens.